Cargo.toml

```diff
@@ -31,6 +31,7 @@ http-body = "0.4.4"
 tonic = {version = "0.7.0", features = ["tls", "tls-roots", "tls-webpki-roots"]}
 prost = "0.10.0"
 tower = { version = "0.4" }
+json = "0.12.4"
```

Comment:
Non-blocking:

`json` and `serde_json` are incompatible. Currently, we're using both, somewhat incoherently...I think serde_json is better supported, with a nicer interface...why are we adding dependencies on json?

Reply:
fixed